src/software/gui/drawing/ball.cpp

```diff
@@ -19,24 +19,29 @@ void drawBallVelocity(QGraphicsScene *scene, const Point &position,
 }
 
 void drawBallPosition(QGraphicsScene *scene, const Point &position,
-                      const double distance_from_ground, const QColor &color)
+                      const double distance_from_ground, QColor color)
 {
+    // A rough estimate of the max distance from the ground the ball will ever reach.
+    // This value sets the maximum above which no change will be visible.
+    const double ball_max_distance_from_ground = 1.25;
+
+    // Decrease the alpha value as the ball moves further from the ground
+    double alpha = normalizeValueToRange<double>(distance_from_ground, 0,
+                                                 ball_max_distance_from_ground, 1.0, 0.4);
+    color.setAlphaF(alpha);
+
+    // Increase the radius of the ball the further from the ground it is.
+    double ball_radius = normalizeValueToRange<double>(
+        distance_from_ground, 0, ball_max_distance_from_ground, BALL_MAX_RADIUS_METERS,
+        6 * BALL_MAX_RADIUS_METERS);
```

Comment:
Could we reduce this to `4`? Right now this gets bigger then a robot, which is a _bit_ much.

Reply:
done